firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -374,36 +373,39 @@ class SearchTests: BaseTestCase {
 
         // Check accessibility
         // swiftlint:disable empty_count
-        try app.performAccessibilityAudit { issue in
-            guard let element = issue.element else { return false }
+        if !iPad() {
+            try app.performAccessibilityAudit { issue in
+                guard let element = issue.element else { return false }
 
-            var shouldIgnore = false
-            // number of tabs in navigation toolbar
-            let isDynamicTypeTabButton = element.label == "1" &&
+                var shouldIgnore = false
```

Comment:
is this the proper indentation?

Reply:
i will have a look in the next PR

Reply:
looked over and yes this is the indentation that xcode is doing